test/RateLimiterStoreAbstract.test.js

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable security/detect-object-injection */
+const { describe, it } = require('mocha');
+const { expect } = require('chai');
+const RateLimiterStoreAbstract = require('../lib/RateLimiterStoreAbstract');
+const RateLimiterRes = require('../lib/RateLimiterRes');
+
+class RateLimiterStoreMemory extends RateLimiterStoreAbstract {
```

Comment:
@evan361425 nice. you're ready to implement new rate limiters after this experience :)